.dockerignore

```diff
@@ -5,3 +5,4 @@
 ./.cache
 ./.dapper
 ./.trash-cache
+./.git/objects/pack
```

Comment:
This folder contains the bulk of the `.git` ~1.6GB size, but is not needed to extract the commit ID or branch when calling `git status` or other git info commands. This enables a much lower copy of only some 40KB to docker containers and still allows use of the `scripts/version.sh`.